R/module-template.R

```diff
@@ -403,122 +409,122 @@ setMethod(
 
-# Usage
+      For help writing in RMarkdown, see http://rmarkdown.rstudio.com/.
 
-```{r module_usage}
-library(SpaDES)
-library(magrittr)
+      # Usage
 
-inputDir <- file.path(tempdir(), \"inputs\") %>% checkPath(create = TRUE)
-outputDir <- file.path(tempdir(), \"outputs\")
-times <- list(start = 0, end = 10)
-parameters <- list(
-  .globals = list(burnStats = \"nPixelsBurned\"),
-  #.progress = list(type = \"text\", interval = 1), # for a progress bar
-  # If there are further modules, each can have its own set of parameters, assigned
-  # as examples below
-  #cropReprojectLccAge = list(useCache = TRUE),
-  #forestSuccessionBeacons = list(
-  #  returnInterval = 1, startTime = times$start,
-  #  .plotInitialTime = times$start, .plotInterval = 1),
-  #forestAge = list(
-  #  returnInterval = 1, startTime = times$start+0.5,
-  #  .plotInitialTime = times$start, .plotInterval = 1),
-  #fireSpreadLcc = list(
-  #  nFires = 3, its = 1e6, drought = 1.2, persistprob = 0, returnInterval = 1,
-  #  startTime = times$start+1, .plotInitialTime = times$start, .plotInterval = 1),
-  #caribouMovementLcc = list(
-  #  N = 1e3, moveInterval = 1, startTime = times$start+1, torus = TRUE,
-  #  glmInitialTime = NA_real_, .plotInitialTime = times$start, .plotInterval = 1)
-)
-modules <- list(\"", name, "\")
-  objects <- list()
-  paths <- list(
-    cachePath = file.path(outputDir, \"cache\"),
-    modulePath = file.path(\"..\"),
-    inputPath = inputDir,
-    outputPath = outputDir
-)
+      ```{r module_usage}
+      library(SpaDES)
+      library(magrittr)
 
-mySim <- simInit(times = times, params = parameters, modules = modules,
-                 objects = objects, paths = paths)
+      inputDir <- file.path(tempdir(), \"inputs\") %>% checkPath(create = TRUE)
+      outputDir <- file.path(tempdir(), \"outputs\")
+      times <- list(start = 0, end = 10)
+      parameters <- list(
+      .globals = list(burnStats = \"nPixelsBurned\"),
+      #.progress = list(type = \"text\", interval = 1), # for a progress bar
+      # If there are further modules, each can have its own set of parameters, assigned
+      # as examples below
+      #cropReprojectLccAge = list(useCache = TRUE),
+      #forestSuccessionBeacons = list(
+      #  returnInterval = 1, startTime = times$start,
+      #  .plotInitialTime = times$start, .plotInterval = 1),
+      #forestAge = list(
+      #  returnInterval = 1, startTime = times$start+0.5,
+      #  .plotInitialTime = times$start, .plotInterval = 1),
+      #fireSpreadLcc = list(
+      #  nFires = 3, its = 1e6, drought = 1.2, persistprob = 0, returnInterval = 1,
+      #  startTime = times$start+1, .plotInitialTime = times$start, .plotInterval = 1),
+      #caribouMovementLcc = list(
+      #  N = 1e3, moveInterval = 1, startTime = times$start+1, torus = TRUE,
+      #  glmInitialTime = NA_real_, .plotInitialTime = times$start, .plotInterval = 1)
```

Comment:
why are these commented lines being added to the template? they are specific to the LCC modules.